### PR TITLE
New build system, Windows support

### PR DESCRIPTION
**Pull request by gtrxAC**

### CODE REVIEW

**Comment by gtrxAC** [comment]
The gitignore has html, js, wasm, data, if you don't plan on adding web support you can remove those.

File: README.md [hunk 1]
```diff
@@ -42,17 +42,26 @@ toheader
...
-$ cd sled/
-$ cd src/
-$ make
-```
 
-## Coming soon
-SLED on Windows.
+### Linux
+1. Run `./setup.sh` to build raylib.
+2. Run `./build.sh` to compile the game.
```
Comment (lauchimoon) [inline README.md]: Be careful, it's not a game, it's a tool :)

File: README.md [hunk 2]
```diff
@@ -42,17 +42,26 @@ toheader
...
 
+### Windows
+1. Download [w64devkit](https://github.com/skeeto/w64devkit/releases):
+* `w64devkit-x.x.x.zip` for 64-bit
+* `w64devkit-i686-x.x.x.zip` for 32-bit
+2. Extract w64devkit and run `w64devkit.exe`.
+3. Inside w64devkit, go to the directory where you cloned sled.
+4. Run `./setup.sh` to build raylib.
+5. Run `./build.sh` to compile the game.
```
Comment (lauchimoon) [inline README.md]: Same issue here.

**Review by lauchimoon — CHANGES_REQUESTED** [review]
All good! Check the review comments before I push these changes.

**Comment by lauchimoon** [comment]
Thanks for the addition!

**Pull request merged**